docs/standards/nft-2.0/LSP8-Identifiable-Digital-Asset.md

```diff
@@ -13,83 +13,105 @@ sidebar_position: 4
+![ERC721 TokenIds Representation](/img/standards/erc721-tokenIds.jpeg)
 
-A good example where all these problems could have effects is a video game. Game services could use an NFT to represent a unique item like a weapon. We might want the player to have the ability to upgrade the weapon features over time so that it becomes more powerful.
+**[LSP8-IdentifiableDigitalAsset](#)** define the tokenIds as **bytes32**. The choice of **bytes32 tokenIds** allows a wide variety of applications including numbers, contract addresses, and hashed values (ie. serial numbers). 
 
-## What does this Standard represent?
+The **bytes32 tokenId** can be interpreted as a <u>**Number**</u>: 
+
+![LSP8 Number TokenIds Representation](/img/standards/lsp8-tokenId-number.jpeg)
+
+
+The **bytes32 tokenId** can be interpreted as a <u>**Hashed Value**</u>: 
 
-LSP8 Identifiable Digital Asset is a standard that aims to describe _non-fungible_ assets. _Non-fungible_ means that each asset is unique and different. They are distinguishable from each other and therefore not interchangeable.
+![LSP8 Hashed Value TokenIds Representation](/img/standards/lsp8-tokenId-hashed.jpeg)
 
-Since every single asset is unique on its own, they are differentiated by a unique identifier: a `tokenId`. The identifier can be anything from a unique serial number to another [ERC725Y](https://github.com/ERC725Alliance/ERC725/blob/main/docs/ERC-725.md#erc725y) smart contract that contains information and metadata specific to this `tokenId`.
 
-LSP8 solves the current problems of NFTs by using the ERC725Y standard as its base. By using a generic data key-value store, an LSP8 contract now comes with the following features:
+The **bytes32 tokenId** can be interpreted as a <u>**Contract Address**</u>: 
 
-- attaching an unlimited amount of information (= metadata), making the asset more customizable and unique.
-- knowing when the metadata has been altered, as a `DataChanged` event will be emitted in ERC725Y.
-- having metadata as a **hash reference** (= data reference), instead of a URL reference.
+![LSP8 Address TokenIds Representation](/img/standards/lsp8-tokenId-address.jpeg)
 
-LSP8 assets are similar to [ERC721](https://eips.ethereum.org/EIPS/eip-721) tokens (NFTs). Their underlying base makes them different in terms of more customization and upgradability of their metadata over time. Such a feature set lets them become part of a new generation of unique assets known as **NFT 2.0**.
+TokenIds represented as **contract address** will allow the creation of more  **complex NFTs**, such as **Nested NFTs**, where each tokenId could be a contract that has its own storage using **[ERC725Y](../lsp-background//erc725.md#erc725y---generic-data-keyvalue-store)**, and the tokenId can **evolve** by changing metadata depending on **game rules**.  
```

Comment:
I am not sure I understand the term _Nested NFTs_.

I removed it in my suggestion, but feel free to keep it or re-word it if it makes sense :)

```suggestion
TokenIds represented as **smart contract address** allow the creation of more  **complex NFTs**. When each tokenId is a contract that with its own **[ERC725Y](../lsp-background//erc725.md#erc725y---generic-data-keyvalue-store)** storage. For instance in a video game, by changing the features and metadata of the NFT based on the **game rules**.  
```